docs/guides/ABCD/fast_track/CHANGES.md

```diff
@@ -4,10 +4,32 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.1.3] - 2024-02-27
+## [1.2.0] - 2024-10-21
+
+### Added
+
+- Provided comprehensive lists of changes in the `code/v1.2.0/` subdirectory for this release including the lists of renamed files, added files, conversion log files, and scripts used to put data in place.
+- Added in 2,207 new sessions (3 baseline, 203 2-year follow-up, 278 4-year follow-up, and 1,723 6-year follow-up). These were converted using a new DSST converter, [nimh-dsst/abcd-fasttrack2bids](https://github.com/nimh-dsst/abcd-fasttrack2bids). The full list of new sessions can be found in `code/v1.2.0/new_sessions.tsv`.
 
 ### Changed
 
+- Replaced all `dwi/` subfolders with those from the new DSST converter to ensure they are complete, correct, and consistent.
```

Comment:
```suggestion
- Replaced all existent `dwi/` folders because they had either incorrect BVAL/BVEC files from the original `DCAN-Labs/abcd-dicom2bids` conversion or they were missing some `dwi` runs.
```